### PR TITLE
Improve warp to destination

### PR DESCRIPTION
**Pull request by Nekaida**

### CODE REVIEW

**Comment by Sierraffinity** [comment]
Could you revert commit c1256c7f6b9e60e8ed44967e586207d201689e2e and redo the PR? Rather than having a message come up, it would be a better idea to have the travel button simply be disabled if pointing at a map that doesn't exist.

File: src/Forms/MainWindow.cpp [hunk 1]
```diff
@@ -1787,6 +1791,30 @@ namespace ame
...
+    void MainWindow::on_warp_bank_valueChanged(int value)
+    {
+        checkWarp();
+    }
+
+    void MainWindow::checkWarp()
+    {
+    if (ui->warp_bank->value() < dat_MapBankTable->banks().size() &&
+        ui->warp_map->value() < dat_MapBankTable->banks().at(ui->warp_bank->value())->maps().size() &&
+        ui->warp_number->value() < dat_MapBankTable->banks()[ui->warp_bank->value()]->maps()[ui->warp_map->value()]->entities().warps().size())
```
Comment (Nekaida) [inline src/Forms/MainWindow.cpp]: would it be better to cache the values in local vars?

**Comment by Nekaida** [comment]
You're right, this way it is more user friendly. Now it checks every value in (hopefully) all relevant situations.
Edit: How much do you care about the comment block above every function?

File: src/Forms/MainWindow.cpp (hunk 1 above)
Comment (Sierraffinity) [reply src/Forms/MainWindow.cpp]: Yeah, that would be best.

**Comment by Sierraffinity** [comment]
We try to fill out the block each time we make a change, so that would be preferred.

**Comment by Nekaida** [comment]
I updated the PR and squashed the commits into one.

**Review by Sierraffinity — APPROVED** [review]

**Pull request merged**